code/modules/holodeck/items.dm

```diff
@@ -50,23 +50,14 @@
 		cards -= card
 		card.forceMove(drop_location())
 
-//BASKETBALL OBJECTS
-
-/obj/item/toy/beach_ball/holoball
-	name = "basketball"
-	icon = 'icons/obj/items_and_weapons.dmi'
-	icon_state = "basketball"
-	inhand_icon_state = "basketball"
-	desc = "Here's your chance, do your dance at the Space Jam."
-	w_class = WEIGHT_CLASS_BULKY //Stops people from hiding it in their bags/pockets
-
-/obj/item/toy/beach_ball/holoball/dodgeball
+/obj/item/toy/beach_ball/dodgeball
 	name = "dodgeball"
 	icon_state = "dodgeball"
 	inhand_icon_state = "dodgeball"
 	desc = "Used for playing the most violent and degrading of childhood games."
+	w_class = WEIGHT_CLASS_BULKY
 
-/obj/item/toy/beach_ball/holoball/dodgeball/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+/obj/item/toy/beach_ball/dodgeball/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
 	..()
 	if((ishuman(hit_atom)))
 		var/mob/living/carbon/M = hit_atom
```

Comment:
single var names bad please change this while you're here

Reply:
(you don't have to, this wasn't your addition https://github.com/tgstation/tgstation/pull/66954)

Reply:
Did it anyways